src/main/views/webContentEvents.js

```diff
@@ -228,7 +228,6 @@ export const addWebContentsEventListeners = (mmview, getServersFunction) => {
     contents.on('page-title-updated', mmview.handleTitleUpdate);
     contents.on('page-favicon-updated', mmview.handleFaviconUpdate);
     contents.on('update-target-url', mmview.handleUpdateTarget);
-    contents.on(FOUND_IN_PAGE, mmview.handleFoundInPage);
```

Comment:
I assume this was the error that caused the refresh before. handleFoundInPage doesn't seem to be defined any more

Reply:
yes, it is this one that was causing the problem. it was catched and retried loading, hence the flashing